src/content/reference/cluster-definition/_index.md

```diff
@@ -2,7 +2,7 @@
 title = "Cluster Definition YAML Reference"
 description = "Complete documentation of the Giant Swarm cluster definition YAML format"
 date = "2017-06-21"
-type = "page"
+layout = "subsection"
 weight = 100
```

Comment:
This applies a specific template (the new `src/layouts/section/subsection.html`) to render this page